.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": [ ">0.25%", "not ie 11", "not op_mini all"],
```

Comment:
Which way would be better? Like this or using `.browserlistrc`

Reply:
I prefer `.browserlistrc` but babel 6 doesn't support it. Babel 7 requires Node.js 6 and thus I ended up here 😆